src/appMain/sdl_preloaded_pt.json

```diff
@@ -3914,6 +3918,30 @@
                     "mandatory": false,
                     "since": "5.1"
                 },
+                {
```

Comment:
@IGapchuk move to the separate commit

Reply:
@IGapchuk all changes already are in separate commit 15cfd4b. Why should I do a separate commit for this particular change?

Reply:
@IGapchuk section moved to commit https://github.com/smartdevicelink/sdl_core/pull/3391/commits/f37eef4b7a7043bcf7e11b7860d41b54a7ec2b4b